### PR TITLE
Update location of published web.config

### PR DESCRIPTION
**Pull request by Tratcher**
Submitting this for John now that Github is back up.

#115

@muratg @davidfowl @DamianEdwards @moozzyk

### CODE REVIEW

**Comment by Tratcher** [comment]
Update the sample too.

**Comment by GuardRex** [comment]
@Tratcher Any guess on the `dotnet cli` version this will hit? ... no worries if not known ... I'll just eyeball my published output in that case until it lands.

**Comment by Tratcher** [comment]
@GuardRex I don't understand, this isn't in the dotnet cli, it's an independent package. We want it in RC2.

File: test/Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests/PublishIISCommandFacts.cs [hunk 1]
```diff
@@ -18,12 +18,11 @@ private class Folders
...
+            var folders = CreateTestDir("{}");
 
-            new PublishIISCommand(folders.PublishOutput, folders.ProjectPath, null).Run();
+            new PublishIISCommand(folders.PublishOutput, folders.ProjectPath).Run();
 
-            var processPath = (string)GetPublishedWebConfig(folders.PublishOutput, webRoot)
+            var processPath = (string)GetPublishedWebConfig(folders.PublishOutput)
                 .Descendants("aspNetCore").Attributes("processPath").Single();
 
             Assert.Equal($@"..\projectDir.exe", processPath);
```
Comment (kichalla) [inline test/Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests/PublishIISCommandFacts.cs]: Now that the web.config file would be a sibling with the executable...this should not have `..`?

**Comment by kichalla** [comment]
Curious, why this change? the bug does not say much

File: test/Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests/PublishIISCommandFacts.cs (hunk 1 above)
Comment (JunTaoLuo) [inline test/Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests/PublishIISCommandFacts.cs]: Ah yes, I forgot to update that.

**Comment by JunTaoLuo** [comment]
@kichalla I believe that's where IIS/Express was expecting it to be located. Currently it will break some scenarios. @Tratcher has more details.

**Comment by GuardRex** [comment]
@Tratcher I see your point ... my bad.

Like @kichalla, I'm just looking for details. IIS will point to the `ContentRootPath` (published content base) directory to pickup the file and config the reverse-proxy. I see the updated paths. Any static content in `wwwroot` is served by static file middleware. Sounds good. I'm going to go ahead and reconfigure my apps and hosting now for these changes unless there is a reason to wait ... ok?

**Comment by kichalla** [comment]
:shipit:

**Comment by Tratcher** [comment]
@kichalla This is one where we'll make @davidfowl write the announcement...

**Comment by JunTaoLuo** [comment]
:up::date: cc @moozzyk

File: samples/IISSample/project.json [hunk 2]
```diff
@@ -9,25 +9,16 @@
...
     "emitEntryPoint": true
   },
-  "commands": {
-    "web": "IISSample"
-  },
   "frameworks": {
     "net451": { },
     "netstandardapp1.5": {
-      "imports": "portable-net451+win8"
+      "dependencies": {
```
Comment (moozzyk) [inline samples/IISSample/project.json]: shouldn't this be `netcoreapp1.0` ?

Comment (JunTaoLuo) [inline samples/IISSample/project.json]: I imagine there's another wave of updates to change that for all samples but I'm not sure when that's happening.

File: test/Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests/WebConfigTransformFacts.cs [hunk 3]
```diff
@@ -99,7 +95,7 @@ public void WebConfigTransform_does_not_remove_children_of_aspNetCore_element()
                 new XElement("environmentVariable", new XAttribute("name", "ENVVAR"), new XAttribute("value", "123"));
 
             var input = WebConfigTemplate;
-            input.Descendants("environmentVariable").Single().Add(envVarElement);
+            input.Descendants("aspNetCore").Single().Add(envVarElement);
```
Comment (moozzyk) [inline test/Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests/WebConfigTransformFacts.cs]: :open_mouth:

File: samples/IISSample/project.json (hunk 2 above)
Comment (moozzyk) [inline samples/IISSample/project.json]: OK

**Comment by moozzyk** [comment]
:ship::it:

**Pull request merged**